Userland/Libraries/LibGfx/ImageFormats/TIFFLoader.cpp

```diff
@@ -18,6 +18,24 @@
 
 namespace Gfx {
 
+namespace {
+
+CCITT::Group3Options parse_t4_options(u32 bit_field)
+{
+    CCITT::Group3Options options {};
+
+    if (bit_field & 0x001)
+        options.dimensions = CCITT::Group3Options::Mode::TwoDimensions;
+    if (bit_field & 0x010)
```

Comment:
This has to use 0b instead of 0x, else it's wrong

Reply:
Thanks for spotting that!